WordPress/src/main/java/org/wordpress/android/util/CrashLogging.kt

```diff
@@ -1,172 +0,0 @@
-package org.wordpress.android.util
-
-import android.content.Context
-import android.preference.PreferenceManager
-import io.sentry.android.core.SentryAndroid
-import io.sentry.core.Sentry
-import io.sentry.core.SentryLevel
-import io.sentry.core.protocol.User
-import org.wordpress.android.BuildConfig
-import org.wordpress.android.R
-import org.wordpress.android.fluxc.store.AccountStore
-import org.wordpress.android.util.AppLog.T
-import org.wordpress.android.util.helpers.logfile.LogFileProvider
-import javax.inject.Inject
-import javax.inject.Singleton
-
-private const val EXTRA_UUID = "uuid"
-private const val EVENT_BUS_MODULE = "org.greenrobot.eventbus"
-private const val EVENT_BUS_EXCEPTION = "EventBusException"
-private const val EVENT_BUS_INVOKING_SUBSCRIBER_FAILED_ERROR = "Invoking subscriber failed"
-
-@Singleton
-class CrashLogging @Inject constructor(
-    private val accountStore: AccountStore,
-    private val encryptedLogging: EncryptedLogging
-) {
-    fun start(context: Context) {
-        SentryAndroid.init(context) { options ->
-            options.dsn = BuildConfig.SENTRY_DSN
-            options.cacheDirPath = context.cacheDir.absolutePath
```

Comment:
If you're be looking into `Automattic-Tracks-Android` sources, we don't set up `cacheDirPath` as Sentry does it on their own: https://github.com/getsentry/sentry-java/blob/6dbed692733c1f5867202b2b6c7ab33ed9f06aeb/sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java#L27